assets/tests/factories/detourStateMachineFactory.ts

```diff
@@ -34,8 +34,15 @@ export const detourInProgressFactory = Factory.define<DetourWithState>(() => {
   }
 })
 
-export const activeDetourFactory = Factory.define<DetourWithState>(() => {
-  // Stub out a detour machine, and start a detour-in-progress
+interface DetourWithStateTransientParams extends DetourWithState {
```

Comment:
question: is the `extend` nessicary?

Reply:
ahhh I assumed that all params were needed in the object, and that's not correct. you're right, omitting!

```suggestion
interface DetourWithStateTransientParams {
```